Documentation/hacking.md

```diff
@@ -105,7 +109,8 @@ Additional build dependencies for the stage1 kvm follow. If building with docker
 rkt is designed and intended to be modular, using a [staged architecture](devel/architecture.md).
 
 `rkt run` determines the stage1 image it should use via its `-stage1-image` flag.
-By default, if this flag is unset at runtime, rkt will default to the configure-time settings. It usually means that rkt will look for a file called `stage1-<default flavor>.aci` that is in the same directory as the rkt binary itself.
+By default, if this flag is unset at runtime, rkt will default to the configure-time settings.
```

Comment:
If this flag is not given to `rkt`, the stage1 image will default to the settings configured when `rkt` was built from source.